.CI/compliance.failures

```diff
@@ -1,7 +1,6 @@
 ModelicaCompliance.Algorithms.For.RealRange
 ModelicaCompliance.Algorithms.For.ShadowedIterator
 ModelicaCompliance.Algorithms.For.StringRange
-ModelicaCompliance.Algorithms.For.VarArrayRange
```

Comment:
@perost Can you check this. I think the test was supposed to fail but was passing. So it was marked as failing. Now it fails so it is not "failure" anymore. Is this right?

The test is this one
https://github.com/modelica/Modelica-Compliance/blob/master/ModelicaCompliance/Algorithms/For/VarArrayRange.mo

It fails at runtime now. With an assert triggered and aborting. I looked at the test and I can imagine it is not trivial to check at flattening.  OF lets it flatten with no issues. NF does not and gives a not implemented notice. 

So I guess bailing at runtime is acceptable for now.

Reply:
Yes, that's correct, the test is supposed to fail. I guess it would be possible for the frontend to catch this in some cases, but I don't think it's possible to catch all cases without actually executing the function.

That the NF fails is unrelated, it doesn't support implicit iteration ranges yet since no one uses them.

Reply:
Okay, the it is all good. 

Just out of curiosity, do you even plan to support this? Does it even have a use case where this would be the only way to iterate over some array instead of specifying it in the iterator range?

Reply:
I'll implement it when someone asks for it, or I don't have anything better to do :)

And I guess the use case would be to iterate over multiple arrays while also making sure they all have the same size, since it should give an error if the dimensions iterated over aren't the same. But it's mostly a convenience feature.

Reply:
> I'll implement it when someone asks for it, or I don't have anything better to do :)
Good plan :) 

I can see it has a use case. It sounds to me like one of those "fancy" Modelica things that sound flexible at first but then end up complicating the "normal" workflow with extra checks and bookkeeping for no-one's gain in the end. 

Good luck :)